Objective-C/CBLReplicator.mm

```diff
@@ -77,11 +78,14 @@ @implementation CBLReplicator
     unsigned _retryCount;
     CBLReachability* _reachability;
     NSMutableSet* _listenerTokens;
+    BOOL _isStopping;
```

Comment:
Would it be clearer to use a state machine (i.e. a `_state` ivar whose type is an enum) rather than multiple booleans? (I don't know the answer, just wondering if you've considered it.)

Reply:
I think the code could look clearer. I will put the TODO: for refactoring the code later after 2.0.